### DISCUSSION TITLE
导入到 Taio

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2506] [插件说明]
描述
---

> 将当前页面以 Markdown 的形式导入到 Taio

地址
---

[https://simpread.ksria.cn/plugins/details/0C7bZGPl4m](https://simpread.ksria.cn/plugins/details/0C7bZGPl4m)

Taio
---

> [Taio](https://taio.app/cn/) 一款在 iPhone、iPad 和 Mac 上进行文本处理的现代化应用，为剪贴板、Markdown 编辑和文本工作流提供无缝的体验。

使用场景
---

> 因为 Taio  可以运行在 Mac 上，所以你可以将导入到其中的 md 做任何事情，并支持 Taio 上的 [动作库](https://actions.taio.app/)。

下载 Taio
---

> 目前 Mac 版的 Taio 正在内测，[下载地址](https://github.com/cyanzhong/Taio-Mac-Beta/releases)。

下载动作
---

> 使用此插件的前提需要下载对应的动作库

[从简悦导入Markdown.taioactions](https://github.com/Kenshin/simpread/files/6789535/Markdown.taioactions.zip)

> 下载并解压缩后，双击 `从简悦导入Markdown.taioactions` 即可自动安装

![image](https://user-images.githubusercontent.com/81074/125040655-7f60d480-e0ca-11eb-9a8b-050f5cd30c8b.png)

> 注意：**保存 .md 位置**为当前目录。

效果
---

![image](https://user-images.githubusercontent.com/81074/125040292-0a8d9a80-e0ca-11eb-83f1-d6690045ab80.png)

![image](https://user-images.githubusercontent.com/81074/125039772-71f71a80-e0c9-11eb-9766-5d7595fa3dbc.png)

> 简悦与 Taio 都可以完美的支持 LaTeX

![image](https://user-images.githubusercontent.com/81074/125064801-9cee6800-e0e3-11eb-9e12-60d3319709f0.png)

功能
---

1. 可通过快捷键 <kbd>t a</kbd> 直接使用
2. 可在阅读模式 → 右下角 → 控制栏 → 插件触发器  → 导入到 Taio 手动触发

更新日志
---

### 0.0.1

1. 支持快捷键 <kbd>t a</kbd>
2. 可在阅读模式 → 右下角 → 控制栏 → 插件触发器  → 导入到 Taio 手动触发

引申
---

> 安装插件或带有插件触发器的插件失效的 [解决方案](https://github.com/Kenshin/simpread/discussions/2342)。

**Comment by kingswon** [comment]
taio有没有md + assets 版本？

**Reply by Kenshin** [reply]
直接使用 https://github.com/Kenshin/simpread/discussions/4041 把路径设置到 Taio 就行了。
